src/player.h

```diff
@@ -554,6 +554,11 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         /** Knocks the player back one square from a tile */
         void knock_back_from( const tripoint &p ) override;
 
+        /** Returns multiplier on fall damage at low velocity (knockback/pit/1 z-level, not 5 z-levels) */
+        float fall_damage_mod() const override;
+        /** Deals falling/collision damage with terrain/creature at pos */
+        virtual int impact( int force, const tripoint &pos ) override;
```

Comment:
`virtual` is redundant with `override` and if you're going to add it anyway, way not at the other virtual function two lines above?